wagtail/project_template/project_name/settings.py

```diff
@@ -137,30 +121,12 @@
 )
 
 
-# Template configuration
+# Template context processors
 
 from django.conf import global_settings
-
 TEMPLATE_CONTEXT_PROCESSORS = global_settings.TEMPLATE_CONTEXT_PROCESSORS + (
     'django.core.context_processors.request',
 )
```

Comment:
> Setting this to a list now doesn't work with Django 1.7/1.8 though.

    TEMPLATE_CONTEXT_PROCESSORS = list(global_settings.TEMPLATE_CONTEXT_PROCESSORS) + [
         'django.core.context_processors.request',
     ]

would cover all bases, wouldn't it?